contents/docs/surveys/installation.mdx

```diff
@@ -7,8 +7,13 @@ availability:
     selfServe: full
     enterprise: full
 ---
-import WebInstall from '../integrate/_snippets/install-web.mdx'
 
-To get started with surveys, first install PostHog in your app:
+> **Important:** When installing PostHog via a package manager, surveys require `posthog-js v1.67+`.
```

Comment:
This needs to be updated with the correct version after Neil removes the dependency on Site apps